Makefile

```diff
@@ -52,6 +52,11 @@ upgrade: ## update the requirements/*.txt files with the latest packages satisfy
 quality: ## check coding style with pycodestyle and pylint
 	tox -e quality
 
+install_mysql: ## install MySQL for running tests.
+	@echo "\n\nSet root password to empty string if asked.\n\n"
```

Comment:
This seems like bad advice.  Maybe it should suggest setting the MYSQL_PASSWORD environment variable to the same value as was chosen for the root user?